src/Compilers/CSharp/Test/EndToEnd/Microsoft.CodeAnalysis.CSharp.EndToEnd.UnitTests.csproj

```diff
@@ -7,7 +7,9 @@
     <TargetFrameworks>net6.0;net472</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
-  <ItemGroup Label="Project References">
+
+  <ItemGroup>
+    <AssemblyAttribute Include="Microsoft.CodeAnalysis.Test.Utilities.RunTestsInSinglePartitionAttribute" />
```

Comment:
Seems like we would either want to generate the attribute or reference the file. Why do we generate the attribute instead of linking to the file?

Reply:
whoops, I was mistaken